### ISSUE TITLE
Powers of complex intervals not working 

### ISSUE THREAD

**Issue by dpsanders** [dpsanders/ValidatedNumerics.jl#141]
Reported by private email:
```
julia> (@interval 1im)^2
ERROR: MethodError: `isinteger` has no method matching isinteger(::ValidatedNumerics.Interval{Float64})
 in ^ at complex.jl:480
 in ^ at complex.jl:538
```
Defining `isinteger` first as 
```
julia> Base.isinteger(x::Interval) = isinteger(x.lo) && x.hi==x.lo
isinteger (generic function with 8 methods)
``` 
gives the following error
```
julia> (@interval 1im)^2
ERROR: MethodError: `convert` has no method matching convert(::Type{Signed}, ::ValidatedNumerics.Interval{Float64})
This may have arisen from a call to the constructor Signed(...),
since type constructors fall back to convert methods.
Closest candidates are:
  call{T}(::Type{T}, ::Any)
  convert(::Type{Signed}, ::UInt8)
  convert(::Type{Signed}, ::UInt16)
  ...
 in ^ at complex.jl:485
 in ^ at complex.jl:538
```

**Comment by dpsanders** [comment]
Note that 
```
julia> a = @interval 1im
[0, 0] + [1, 1]*im

julia> a*a
[-1, -1] + [0, 0]*im
```
does work.

**Comment by dpsanders** [comment]
Fixed by #142
